docs/en/integrations/index.asciidoc

```diff
@@ -0,0 +1,915 @@
+
+==== Add screenshots
+
+The Kibana Integration Manager shows screenshots related with the integration. Screenshots present Kibana dashboards visualizing the metric/log data.
+
+==== Create a readme file
+
+The README template is used to render the final README file including exported fields. The template should be placed in the package/<integration-name>/_dev/build/docs/README.md. If the directory doesn't exist, please create it.
+
+Review the MySQL docs template to see how to use template functions (e.g. {{fields "data-stream-name"}}). If the same data stream name is used in both metrics and logs, please add -metrics and -logs in the template. For example, elb is a data stream for log and also a data stream for metrics. In README.md template, {{fields "elb_logs"}} and {{fields "elb_metrics"}} are used to separate them.
+
+==== Review artifacts
+
+==== Define variable properties
+
+The variable properties customize visualization of configuration options in the Kibana UI. Make sure they're defined in all manifest files.
+
+[source,yaml]
+----
+vars:
+  - name: paths
+    required: true <1>
+    show_user: true <2>
+    title: Access log paths <3>
+    description: Paths to the nginx access log file. <4>
+    type: text <5>
+    multi: true <6>
+    default:
+      - /var/log/nginx/access.log*
+----
+<1> option is required
+<2> don't hide the configuration option (collapsed menu)
+<3> human readable variable name
+<4> variable description (may contain some details)
+<5> field type (according to the reference: text, password, bool, integer)
+<6> the field has mutliple values.
+
+==== Add sample events
+
+=== Promote (?)
+
+// https://github.com/elastic/integrations/blob/master/docs/developer_workflow_promote_release_integration.md
+
+=== Open a PR
+
+If you think that you've finished works on your integration, you've verified that it collects data, wrote some tests, you can open a PR to include your integration in the Integrations repository. The CI will verify if your integration is correct (elastic-package check) - a green status is a must.
+
+// When the PR is merged, the CI will kick off a build job for the master branch, which can release your integration to the package-storage. It means that it will open a PR to the Package Storage/snapshot with the built integration if only the package version doesn't already exist in the storage (hasn't been released yet).
+
+// When you are ready for your changes in the integration to be released, remember to bump up the package version. It is up to you, as the package developer, to decide how many changes you want to release in a single version. For example, you could implement a change in a PR and bump up the package version in the same PR. Or you could implement several changes across multiple PRs and then bump up the package version in the last of these PRs or in a separate follow up PR.
+
+== Dashboards
+
+sooooooon
```

Comment:
https://github.com/elastic/integrations/issues/269 a lot of content can be taken from this issue